src/components/widgets/ComboboxDepartment.vue

```diff
@@ -173,6 +174,28 @@ export default {
       } else {
         return this.departmentList[0]
       }
+    },
+
+    listStyle() {
+      const data = {
+        'max-height': this.maxHeightSelectInput + 'px',
+        width: this.width + 'px',
+        top: this.top ? '30px' : '37px',
+        left: '0px'
+      }
+      if (this.top) {
+        Object.assign(data, {
+          top: '-200px',
+          bottom: '-90px',
+          'border-top-left-radius': '10px',
+          'border-top-right-radius': '10px',
+          'border-bottom-left-radius': '0px',
+          'border-bottom-right-radius': '0px'
+        })
```

Comment:
The `top` value is set twice:
- line "183" : IF "top" THEN 30px ELSE 37px
- line "188" : IF "top" THEN -200px

The "30px" will never be applied.

```suggestion
      const data = {
        'max-height': this.maxHeightSelectInput + 'px',
        width: this.width + 'px',
        top: '37px',
        left: '0'
      }
      if (this.top) {
        Object.assign(data, {
          top: '-200px',
          bottom: '-90px',
          'border-top-left-radius': '10px',
          'border-top-right-radius': '10px',
          'border-bottom-left-radius': '0',
          'border-bottom-right-radius': '0'
        })
```